charts/keycloak/values.yaml

```
@@ -1 +1 @@
----
+---
```

```suggestion
---

```

Please leave the white line at the end of the file :wink: